docs/user_guides/fs/tags/tags.md

```diff
@@ -0,0 +1,138 @@
+# Tags 
+
+## Introduction
+
+Hopsworks feature store enables users to attach tags to artifacts, such as feature groups, feature views or training datasets.
+
+A tag is a `{key: value}` association which provides additional information about the data managed by Hopsworks. Tags allow organizations to customize the feature store metadata and build a governance layer on top of it. Hopsworks indexes tags and makes them searchable by users on the platform, increasing the discoverability of features and artifacts in the feature store.
```

Comment:
A tag is a `{key: value}` pair

Reply:
Tags allow you to design custom metadata for your artifacts. For example, you could design a tag schema that encodes governance rules for your feature store, such as classifying data as personally identifiable, defining a data retention period for the data, and defining who signed off on the creation of some feature.